docs/building-spark.md

```diff
@@ -111,7 +111,7 @@ should run continuous compilation (i.e. wait for changes). However, this has not
 extensively. A couple of gotchas to note:
 
 * it only scans the paths `src/main` and `src/test` (see
-[docs](http://scala-tools.org/mvnsites/maven-scala-plugin/usage_cc.html)), so it will only work
+[docs](http://davidb.github.io/scala-maven-plugin/example_compile.html)), so it will only work
```

Comment:
@sarutak, what do you think about using http://davidb.github.io/scala-maven-plugin/example_cc.html instead? (I [checked it](https://github.com/davidB/scala-maven-plugin/blob/2.9.1/src/site/xdoc/usage_cc.xml) although it looks not a big deal though as they have similar information).

Reply:
I confirmed [Internet Archive](https://web.archive.org/web/20160314050540/http://scala-tools.org/mvnsites/maven-scala-plugin/usage_cc.html) and I found the link you suggested is more proper. I'll modify it soon. Thanks.